packages/autocomplete-client/scripts/update_version.js

```diff
@@ -1,6 +1,6 @@
 const pkg = require('../package.json')
-const fs = require('fs')
-const path = require('path')
+const fs = require('node:fs')
```

Comment:
Why do you have to make this change? 👀

Reply:
It's the modern way of importing built-in Node.js modules. The `node:` specifier makes it immediately obvious and non-ambiguous that it's a node built-in module, my IDE complained that `Coding assistance for Node.js is disabled`, so I added this prefix and the warning disappeared.

I'm sorry if I'm making it harder to review these changes, just thought I'd improve a little thing here and there if I'm going to contribute. I can revert these changes if you want to.

Some useful info:
- https://nodejs.org/api/esm.html#node-imports
- https://2ality.com/2021/12/node-protocol-imports.html
- https://docs.deno.com/runtime/manual/node/node_specifiers